AmazonChimeSDKDemo/AmazonChimeSDKDemo/MeetingViewController.swift

```diff
@@ -58,6 +58,9 @@ class MeetingViewController: UIViewController {
 
     // Model
     var meetingModel: MeetingModel?
+    
+    // isLocalOn
+    private var isLocalOn: Bool = false
```

Comment:
Nit: rename to `wasLocalVideoOn`? Also, do not just repeat the variable name as a comment 😊

Reply:
Updated the name and moved to `MeetingModel`

Reply:
Forgot to delete this?